cmd/wksctl/profile/enable/enable.go

```diff
@@ -87,3 +108,26 @@ func profileEnableRun(params profileEnableFlags) error {
 
 	return nil
 }
+
+func processIgnoreFile(dir string, reader io.Reader) ([]string, error) {
```

Comment:
`parseIgnoreFile` would be a more meaningful name

Reply:
changed